starter-code 6/js/index.js

```diff
@@ -0,0 +1,53 @@
+// Variables
+var dice = 1;
+var displaySquares = document.getElementsByClassName('square');
+var display5 = document.getElementsByClassName('five');
+
+// Functions
+
+// Run before every roll to reset the display
+function clearDisplay() {
+  for(var i = 0; i < displaySquares.length; i++) {
+   displaySquares[i].style.backgroundColor="white";
+  }
+  rollDice();
+}
+
+function rollDice() {
+  dice = Math.floor(Math.random() * 6) + 1;
+  console.log(dice); // For verification
+  displayResult();
+}
+
+function displayResult() {
+  switch(dice) {
```

Comment:
This function is too big! A function should do one small job and you combine a bunch together to perform a big task.